### ISSUE TITLE
ARIA Live region (aria-live="polite"): image's alt texts are not announced by screen reader

### ISSUE THREAD

**Issue by jmuheim** [hakimel/reveal.js#3757]
I think it's wonderful that Reveal.js announces slide contents to screen readers using an `aria-live` region. I first thought that the whole slide is such a region, but then I realised that there's a dedicated visually hidden `<div aria-live="polite">` which is fed with the current slide's contents manually. Fair enough, as long as it works, this looks solid and reasonable!

But then I realised that images aren't announced at all through this mechanism. Here the `alt` value is "Abbildung 2":

![Image](https://github.com/user-attachments/assets/0388f63d-472e-4a19-b994-f34b61a6e827)

But this isn't fed to the ARIA Live region:

![Image](https://github.com/user-attachments/assets/0004f33c-27b3-46ec-a5ac-a98fad3648ea)

Probably alt-texts aren't treated as "real" content, which is a shame. The same is the case for `<iframe>`'s titles.

**Comment by jmuheim** [comment]
Great, thank you very much!
